UPGRADING.md

```diff
@@ -3,6 +3,24 @@
 This guide provides steps to be followed when you upgrade your applications to
 a newer version of Tendermint Core.
 
+## v0.31.0
+
+Since the pubsub no longer blocks on sending, some WS clients might stop working as expected. 
+If your WS client is not consuming events fast enough, Tendermint can terminate the subscription. 
+In this case, the WS client will receive an error with description:
```

Comment:
ref: https://github.com/tendermint/tendermint/pull/3417#issuecomment-472285393

There is not much people can do about this. It is merely to inform people, right? @melekes

Reply:
Right. Normally they should not see that error.